src/databricks/labs/remorph/transpiler/execute.py

```diff
@@ -48,7 +50,7 @@ def process_file(config: MorphConfig, input_file: str | Path, output_file: str |
                     w.write(output)
                     w.write("\n;\n")
                 else:
-                    validate = Validate(config.sdk_config)
+                    validate = Validate(WorkspaceClient(config=config.sdk_config))
```

Comment:
Inject workspace client through the function argument, it's available in the cli. Every workspace client must have product name and product version for tracking